solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -46,10 +78,30 @@ function download_deploy_script(){
 #Install yq 4.13.3
 
 function install_yq(){
-    pip3 show yq && pip3 uninstall yq -y
-    wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY}.tar.gz -O - | tar xz && mv ${YQ_BINARY} /usr/bin/yq
-    if [ -f /usr/local/bin/yq ]; then rm -rf /usr/local/bin/yq; fi    
-    ln -s /usr/bin/yq /usr/local/bin/yq
+    try
```

Comment:
This will throw an exception if only the last command is failed or any of the commands in the function fails. We already had command-level exception handling using `throw $Exception`. Can we use that.

Reply:
No. you can put exception handling on any command using `throw $Exception` but those commands should come under try block to get catch. nothing like only last command failure will throw exception.

Reply:
So we need to add `|| throw $Exception` for every command In addition try catch block to make it work for all commands?